CLAUDE.md

```
@@ -0,0 +1,189 @@
+
+**English Only** - This is an open-source project. All contributions must be in English:
+
+- Commit messages
+- Pull request titles and descriptions
+- Code comments
+- Documentation
+- Issue reports and discussions
+
+### Git Workflow
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)